app/views/courses/_about_schools.html.erb

```diff
@@ -1,13 +1,17 @@
 <div class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-l" id="section-schools"><%= course.placements_heading %></h2>
   <div data-qa="course__about_schools">
+  <% if course.program_type == 'higher_education_programme' %>
     <%= render AdviceComponent.new(title: 'Where you will train') do %>
-      <% if course.provider_type == 'university' %>
-        <%= render partial: 'courses/placement/hei', locals: { course: course } %>
-      <% else %>
-        <%= render partial: 'courses/placement/scitt', locals: { course: course } %>
-      <% end %>
+      <%= render partial: 'courses/placement/hei', locals: { course: course } %>
     <% end %>
+  <% elsif course.program_type == 'scitt_programme' %>
+    <%= render AdviceComponent.new(title: 'Where you will train') do %>
+      <%= render partial: 'courses/placement/scitt', locals: { course: course } %>
+    <% end %>
+  <% end %>
+  <% if course.how_school_placements_work.present? %>
```

Comment:
Given the course page already checks for the presence of `course.how_school_placements_work`, not sure needed here as well.

Reply:
there's a situation where we end up wanting to render scitt and HE guidance, but `how_school_placements_work = nil`. In this situation, this method will blow up